docs/install.rst

```diff
@@ -12,12 +12,9 @@ Installation
 Next steps
 ~~~~~~~~~~
 
-- Checkout `beta.mxnet.io <http://beta.mxnet.io/install/index.html>`_ for more options such as ARM devices and docker images.
```

Comment:
Could you update the links instead of deleting them?

Reply:
Uh... that's what the original PR did but y'all said there was too much MXNet.

Reply:
I think we can probably keep crush course.